ARIAKA.SmartGourmet.DATA/Migrations/Configuration.vb

```diff
@@ -2,10 +2,11 @@ Imports System
 Imports System.Data.Entity
 Imports System.Data.Entity.Migrations
 Imports System.Linq
+Imports ARIAKA.SmartGourmet.DATA.Model
```

Comment:
Por que se agrego esta referencia aca??

Reply:
> Por que se agrego esta referencia aca??

Porque me tiraba error en la línea 10 de ese mismo archivo "Inherits DbMigrationsConfiguration(Of SGContext)", diciendo que no reconocía SGContext, y que podía solucionarlo agregando la referencia, o llamandolo "(Of Model.SGContext)", aunque ahora lo probé sin la referencia y funciona igual, así que lo remuevo nomas.

Reply:
> Por que se agrego esta referencia aca??

No, de hecho me sigue saliendo el error, por eso lo puse.

Reply:
ok, usalmente cuando es solamente una referencia agregamos el namespace antes del nombre de la clase que se invoca. Es decir deberia decir  ´´´Model.SGContext´´´

Reply:
Cambio hecho.